src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -7,8 +7,8 @@ All our JavaScript-related SDKs provide the same API. Still, there are some diff
 
 The quickest way to get started is to use the CDN hosted version of the JavaScript browser SDK:
 
-``` javascript
-<script src="https://browser.sentry-cdn.com/5.4.0/bundle.min.js" crossorigin="anonymous"></script>
+```html
+<script src="https://browser.sentry-cdn.com/{% sdk_version sentry.javascript.browser %}/bundle.min.js" crossorigin="anonymous"></script>
```

Comment:
Please make sure to always use the placeholder in appropriate places since it makes sure we are not showing an old version to users.

Reply:
Yes. Thanks!